features/aggregating_failures.feature

```diff
@@ -0,0 +1,97 @@
+@wip @announce
+Feature: Aggregating Failures
+
+  Normally, an expectation failure causes the example to immediately bail.  Sometimes, however, you have multiple, independent expectations, and you'd like to be able to see all of the failures rather than just the first.  One solution is to split off a separate example for each expectation, but if the setup for the examples is slow, that's going to take extra time and slow things down. `aggregate_failures` provides an alternate solution. Within the block, expectation failures will not abort the example like normal; instead, the failures will be aggregated into a single exception that is raised at the end of the block, allowing you to see all expectations that failed.
+
+  `aggregate_failures` takes an optional string argument that will be used in the aggregated failure message as a label.
+
+  Scenario: Multiple Expectation Failures in the same example are all reported
```

Comment:
> If a cuke scenario is very end-to-end, it can do so many things in a step that aggregating failures at step level won't make sense.

Maybe.  After this is released I'll probably reach out to the cucumber team and let them know the feature is available and could be used to wrap each step and it's up to them to decide if that wold help meet user needs or not.